src/Annotation/EloquaAppCloudActionResponder.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\eloqua_app_cloud\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Eloqua AppCloud Decision Responder item annotation object.
+ *
+ * @see \Drupal\eloqua_app_cloud\Plugin\EloquaAppCloudDecisionResponderManager
+ * @see plugin_api
+ *
+ * @Annotation
+ */
+class EloquaAppCloudActionResponder extends Plugin {
+
+
+  /**
+   * The plugin ID.
+   *
+   * @var string
+   */
+  public $id;
+
+  /**
+   * The label of the plugin.
+   *
+   * @var \Drupal\Core\Annotation\Translation
+   *
+   * @ingroup plugin_translatable
+   */
+  public $label;
+
+  /**
+   * The list of fields required by the plugin.
+   *
+   * @var array
+   *
+   */
+  public $fieldList;
+
+  /**
+   * The API type (contacts or customObject).
+   * @var string
+   */
```

Comment:
API seems to be what it is called internal to the elomentary client, so that is what I called it here.
https://github.com/tableau-mkt/elomentary/blob/c9e8e67f8239cfd813fae59fe665ed7130c439d2/src/Client.php#L60

That said, I see your point, how about client_api?